CONTRIBUTING.md

```diff
@@ -74,7 +74,7 @@ We don't officially support building from source using `pip`, but _if_ you do, y
 #### Other development dependencies (some of these are needed to run tests):
 
 ```
-pip install expecttest flake8 typing mypy pytest pytest-mock scipy
+pip install expecttest flake8 typing mypy pytest pytest-mock scipy pillow
```

Comment:
That shouldn't be needed here as it's already a hard-dependency of torchvision, so it should get installed when you install torchvision from source.

Curious if you encounter any issue here?

Reply:
hmmm odd. I remember having to install it manually after building from source, as I got a package not installed error from the tests. I'm not familiar with the internals of `python setup.py develop`, but I can say that I've used `pip install -e .` instead before. There is [some discussion on SO ](https://stackoverflow.com/questions/19048732/python-setup-py-develop-vs-install)about dependencies not being handled correctly with the former, but I'm not sure how much weight to give that.

Happy to remove this though, I may have just done something odd / could be a personal configuration issue

Reply:
Thanks for the info - let me just remove this so we can move forward with this PR and if we're getting more reports about this I'll investigate. Thank you!